sql/item_mods.sql

```diff
@@ -19998,16 +19998,40 @@ INSERT INTO `item_mods` VALUES (16397,25,3);
 INSERT INTO `item_mods` VALUES (16398,25,3);
 INSERT INTO `item_mods` VALUES (16398,431,1);
 INSERT INTO `item_mods` VALUES (16403,431,1);
+INSERT INTO `item_mods` VALUES (16404,431,1);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,499,10);  -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,501,10);  -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,951,3);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,952,4);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,953,30);  -- Venom Katars - Additional effect: Poison
 INSERT INTO `item_mods` VALUES (16410,431,1);
 INSERT INTO `item_mods` VALUES (16416,54,5);
+INSERT INTO `item_mods` VALUES (16416,431,1);   -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,499,1);   -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,500,30);  -- Dragon Claws - Additional effect: Fire damage
```

Comment:
need 
```sql
ITEM_ADDEFFECT_ELEMENT  = 950, -- Element of the Additional Effect or Spikes, for resist purposes
```
so that the script knows to check against fire resists when this damage is dealt, otherwise your 30 damage set  here will never have any resists

Reply:
same is true for all add effects that deal elemental damage